apps/system/js/media_playback.js

```diff
@@ -185,6 +190,13 @@ MediaPlaybackWidget.prototype = {
   updatePlaybackStatus: function mp_updatePlaybackStatus(status) {
     switch (status.playStatus) {
       case 'PLAYING':
+        // WORKAROUND: Sometimes we can still get the playing event after music
```

Comment:
nit: Since it's a unfortunate workaround it should be with a bug number indicates to the real solution. Although I don't know for this case which bug should be indicated.